src/plans/ID3MPlan.sol

```diff
@@ -16,11 +16,28 @@
 
 pragma solidity 0.6.12;
 
+/**
+    @title D3M Plan Interface
+    @notice Plan contracts are the contract that the Hub uses to determine how much to change its position
+*/
 interface ID3MPlan {
     event Disable();
 
-    // --- Plan Functions ---
+    /**
+        @notice Determines what the position should be based on current assets and the
```

Comment:
Why is there a line break in the middle of this comment but in other (sometimes) longer comments not?